SmartDeviceLink/SDLLockScreenPresenter.m

```diff
@@ -39,38 +37,53 @@ - (instancetype)init {
 }
 
 /// Resets the manager by dismissing and destroying the lockscreen.
-- (void)stop {
+- (void)stopWithCompletionHandler:(nullable SDLLockScreenDidFinishHandler)completionHandler {
     self.shouldShowLockScreen = NO;
 
     if (self.lockWindow == nil) {
         return;
     }
```

Comment:
```suggestion
    if (self.lockWindow == nil) {
        if (completionHandler != nil) {
            completionHandler();
        }
        return;
    }
```

Reply:
When `stopWithCompletionHandler:` is called from `SDLLockScreenPresenter`'s `start` method and it returns here before calling the `completionHandler`, then `sdl_start` is never called in `SDLLockScreenPresenter`. This means that `canPresent` is forever stuck at `NO`. 

## Reproduction steps
I can reproduce this issue with the LMP on iOS 13.3.

1. Open LMP on the device. 
1. Set the TDK moving status to "Speed"
1. Connect LMP to the TDK with a USB cord.
1. Launch the app on the TDK by selecting the app icon. 

### Expected result
Lockscreen shows because the TDK's moving status is set to "Speed".

### Actual result
Lockscreen never shows, even when I toggle the moving status button.